trade-finance-manager-ui/templates/utilisation-reports/utilisation-report-reconciliation-for-report.njk

```diff
@@ -34,11 +34,7 @@
 
     <div class="govuk-grid-row premium-payments-description-filter-container">
       <div class="govuk-grid-column-one-half premium-payments-description-container">
-        <p class="govuk-body">Received payments are entered against reported fees through selection and then selection of the 'Add a payment' button.</p>
-
-        <p class="govuk-body">
-          When payments show as matched, the adjustment data for keying into ACBS will be automatically generated when the 'Generate keying data' button is selected.
-        </p>
+        <p class="govuk-body" data-cy="received-payments-text">Received payments are entered against reported fees. When payments show as matched, the adjustment data for keying into ACBS will be automatically generated.</p>
```

Comment:
Should this change not be for only the read only users with PDC_RECONCILE still seeing the same text?